modules/install/pages/install.adoc

```diff
@@ -44,23 +61,165 @@ Brief blurb about uninstall.....
 
 == References
 
-* Links to all relevant product docs
+* https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/8.0/html/red_hat_jboss_enterprise_application_platform_installation_methods/index[Red Hat JBoss Enterprise Application Platform Installation Methods]
+
 
 == Lab 1: Installing JBoss EAP using the Zip file
 
-. Step 1
-. Step 2
-.. Step 2.a
-.. Step 2.b
-. Step 3
+* Install OpenJDK 17 if not already installed:
++
+[subs="+quotes,+macros"]
+----
+[root@localhost ~]# sudo yum install java-17-openjdk -y
+----
+
+* Download the JBoss EAP zip file from https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?product=appplatform&downloadType=distributions[`Red Hat JBoss Enterprise Application Platform 8.0 Installation Manager`].
+
+* Copy the zip file under `/opt`.
++
+[subs="+quotes,+macros"]
+----
+[root@localhost ~]# cp ~/Downloads/jboss-eap-8.0.0-installation-manager.zip /opt
+----
+
+* Swtich to /opt and uncompress the zip file:
++
+[subs="+quotes,+macros"]
+----
+[root@localhost opt]# unzip /opt/jboss-eap-8.0.0-installation-manager.zip
+----
+
+* Switch to the uncompressed folder:
++
+[subs="+quotes,+macros"]
+----
+[root@localhost opt]# cd jboss-eap-installation-manager-1.1.6.Final-redhat-00001/bin
+----
+
+* Install JBoss EAP by running the following command:
++
+[subs="+quotes,+macros"]
+----
+[root@localhost bin]# ./jboss-eap-installation-manager.sh install --profile eap-8.0 --dir eap-8
+Installing profile: eap-8.0
+Using channels:
+# eap-8.0
+  manifest: org.jboss.eap.channels:eap-8.0
+_...output omitted..._
+----
++
+Review the end user license agreement, and follow the rest of the prompts to install JBoss EAP.
++
+[subs="+quotes,+macros"]
+----
+Accept the agreement(s) [y/N]y
+Feature-packs resolved.
+Packages installed.
+Downloaded artifacts.
+JBoss modules installed.
+_...output omitted..._
+----
+
+* Start the JBoss EAP using the start script present under `/opt/jboss-eap-installation-manager-1.1.6.Final-redhat-00001/bin/eap-8/bin/standalone.sh`
++
+[subs="+quotes,+macros"]
+----
+[root@localhost ~]# /opt/jboss-eap-installation-manager-1.1.6.Final-redhat-00001/bin/eap-8/bin/standalone.sh
+----
+
+* Navigate to the JBOSS console at http://localhost:8080
```

Comment:
We discussed this - for RHEL 9 VMs replace localhost with the NAT IP address of Linux VM.